docs/templates/shared/_head.html.j2

```diff
@@ -2,6 +2,11 @@
     <meta charset="utf-8">
     <title>{{ headline }}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:title" content="{{ headline }}">
+    <meta property="og:description" content="jq is a lightweight and flexible command-line JSON processor">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="{{ url }}/icon.png">
```

Comment:
`{{ url }}{{ root }}/icon.png` I think; root is empty by default, and the icon is in `{{ root }}/icon.png` below

Reply:
Ah true, fixed and generated didn't change so seems it's empty

Reply:
Oh sorry i'm confused. Looks correct now?